backend/Models/Database/Sightings.cs

```diff
@@ -0,0 +1,19 @@
+using System; 
+
+namespace WhaleSpotting.Models.Database
+{
+    public class Sightings
+    {
+        public int Id { get; set; }
+        public int WhaleSpeciesId { get; set; }
+        public string SeenBy { get; set; }
+        public DateTime SeenOn { get; set; }
+        public string ImageUrl { get; set; }
+        public string Description { get; set; }
+        public int WhaleCount { get; set; }
+        public int ConfirmationStatusId { get; set; }
+        public int LocationId { get; set; }
```

Comment:
Remember, we're using Entity Framework, and these joins are done by storing actual references to the classes:
```suggestion
        public ConfirmationStatus ConfirmationStatus { get; set; }
        public Location Location { get; set; }
```